### PR TITLE
Consumer API SDK: BaseAddress has Backslashes

### PR DESCRIPTION
**Pull request by MH321Productions**
# Readiness checklist

-   [ ] I added/updated unit tests.
-   [ ] I added/updated integration tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

### CODE REVIEW

File: BuildingBlocks/src/BuildingBlocks.SDK/UriUtils.cs [hunk 1]
```diff
@@ -2,6 +2,5 @@
 
 public class UriUtils
 {
-    public static Uri CreateUriFromParts(params string[] parts)
-        => new Uri(parts.Aggregate((a, b) => $"{a.TrimEnd('/')}/{b.TrimStart('/')}"));
+    public static Uri CreateUriFromParts(params string[] parts) => new(parts.Aggregate((a, b) => $"{a.TrimEnd('/')}/{b.TrimStart('/')}"));
```
Comment (tnotheis) [inline BuildingBlocks/src/BuildingBlocks.SDK/UriUtils.cs:5]: Should be extracted into a method. Currently it's very hard to read and understand

**Review by tnotheis — CHANGES_REQUESTED** [review]
A few comments about the title of the PR:
1. This doesn't apply to the Consumer API. It's just the SDK.
As it currently is it would show up in the bugs in the Consumer API changelog. And that's just not true.
2. Bugs should always be formulated like you would formulate a bug report. This means: describe what the user sees.

File: BuildingBlocks/src/BuildingBlocks.SDK/UriUtils.cs (hunk 1 above)
Comment (MH321Productions) [reply BuildingBlocks/src/BuildingBlocks.SDK/UriUtils.cs:5]: Would it be better like this?
```c#
var path = parts.Aggregate((a, b) => string.Format("{0}/{1}", a.TrimEnd('/'), b.TrimStart('/')));
```

Or with the interpolated string from earlier (bc ReSharper recommends me to use it)

Comment (tnotheis) [reply BuildingBlocks/src/BuildingBlocks.SDK/UriUtils.cs:5]: Hmm, I'm okay with both.

**Pull request merged**